<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="7" hostname="LAPTOP-5RT7MAM2" failures="1" tests="2" name="Test(failed)" time="68.977" errors="0" timestamp="2023-01-07T13:03:41 IST">
  <testcase classname="Testcases.OverViewPageTest" name="titleofViewPageTest" time="0.112"/>
  <testcase classname="Testcases.FailedTest" name="Test4" time="0.261">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at Testcases.FailedTest.Test4(FailedTest.java:46)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- Test4 -->
  <testcase classname="Testcases.OverViewPageTest" name="paymentInfoTest" time="0.0">
    <ignored/>
  </testcase> <!-- paymentInfoTest -->
  <testcase classname="Testcases.FailedTest" name="Test1" time="0.0">
    <ignored/>
  </testcase> <!-- Test1 -->
  <testcase classname="Testcases.FailedTest" name="Test5" time="0.0">
    <ignored/>
  </testcase> <!-- Test5 -->
  <testcase classname="Testcases.OverViewPageTest" name="ShippingInfoTest" time="0.0">
    <ignored/>
  </testcase> <!-- ShippingInfoTest -->
  <testcase classname="Testcases.FailedTest" name="Test2" time="0.0">
    <ignored/>
  </testcase> <!-- Test2 -->
  <testcase classname="Testcases.OverViewPageTest" name="FinishButtonTest" time="0.0">
    <ignored/>
  </testcase> <!-- FinishButtonTest -->
  <testcase classname="Testcases.FailedTest" name="Test3" time="0.0">
    <ignored/>
  </testcase> <!-- Test3 -->
</testsuite> <!-- Test(failed) -->
